pkg/serverconfig/types.go

```diff
@@ -0,0 +1,56 @@
+package serverconfig
+
+// Config is the top-level console configuration.
+type Config struct {
+	APIVersion    string `yaml:"apiVersion"`
+	Kind          string `yaml:"kind"`
+	ServingInfo   `yaml:"servingInfo"`
+	ClusterInfo   `yaml:"clusterInfo"`
+	Auth          `yaml:"auth"`
+	Customization `yaml:"customization"`
+	Providers     `yaml:"providers"`
+}
+
+// ServingInfo holds configuration for serving HTTP.
+type ServingInfo struct {
+	BindAddress string `yaml:"bindAddress"`
```

Comment:
Shall we add the `omitempty ` as we did in https://github.com/openshift/console-operator/blob/master/pkg/console/subresource/consoleserver/config.go ?

Reply:
Good call, I'll update.